src/components/InventoryGroupDetail/NoSystemsEmptyState.js

```diff
@@ -30,6 +30,7 @@ const NoSystemsEmptyState = ({ groupId, groupName }) => {
         setIsModalOpen={setIsModalOpen}
         groupId={groupId}
         groupName={groupName}
+        edgeParityIsAllowed={true}
```

Comment:
Maybe it is better to provide a feature flag here? It will help us releasing the app without hesitations

Reply:
Feature flag option is not realisable here , as the AddSystemsToGroupModal component is used in two different places and we want to allow edge parity only when AddSystemsToGroupModal component is used in group details empty state